### ISSUE TITLE
I.S. missed operator recruitments

### ISSUE THREAD

**Issue by Constrat** [MaaAssistantArknights/MaaAssistantArknights#8572]
### Description
From #8524

```json
"RoguelikeRecruitLevel": {
        "rectMove": [
            -177,
            -7,
            23,
            16
        ]
    },
```
After adding this, to fix the level recognition, the recruiting algorithm seems off.
MAA starts detecting operators, elites and levels. For level 1 operators, the detection is completely off and no result is shown (who cares, they are under levelled). After this, instead of swiping for other operators (4★), MAA decides to completely skip the recruitment.
The skip is done without knowing the entire operator roster (4★ and 3★, in this specific example `卡达Click` is a very good operator, which was skipped).

### Version

https://github.com/MaaAssistantArknights/MaaAssistantArknights/tree/feat/IS4-2
This was happening all the way to the latest commit and most likely happening all the way to dev-HEAD 47f93548fc8a9777dd6db2b94d6a775b851ba6fa

### Log files

Extract of the issue.
[log.log](https://github.com/MaaAssistantArknights/MaaAssistantArknights/files/14551764/log.log)

**Comment by Manicsteiner** [comment]
https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues/8551
I think this is caused by c++ source code changes, just replacing MaaCore.dll with an older version (v5.2.0-beta.1.d032.g9da3312a2) will fix it

**Comment by Lancarus** [comment]
maa only recognize levels and recruit operators whose level is more than 55. So even the opera is elite 2，level 33，he also can't be recruited

**Comment by Manicsteiner** [comment]
> Btw, with the older .DLL, was the level recognition correct? Because I had to manually modify it for EN and H3XN was testing it for me in KR and CN

This kind of thing did not happen in the lower version, but it happened in the higher version. I thought I should try modifying RoguelikeRecruitLevel roi as well.

**Comment by Constrat** [comment]
@Manicsteiner Considering the amount of issues it created... The "roi" fix is just a patch and not a full fix. The ROI I added seems to be working only for the "starting" recruit. As you can see in very other instance of recruiting (e.g. after a fight) MAA won't swipe... Maybe we should revert the Core change? Or at least understand why it broke everything, considering this seems to be ALSO an Official server issue.

**Comment by Constrat** [comment]
![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/56174894/93665f2e-f420-4cce-9130-9f890544ad05)
The only thing I'm extracting is this:

**Comment by Constrat** [comment]
Can definitely confirm that using MaaCore.dll from `v5.2.0-beta.1.d032.g9da3312a2` WITHOUT `RoguelikeRecruitLevel` change works as expected and like it used to.

**Comment by Constrat** [comment]
Fixed in [f90fc74](https://github.com/MaaAssistantArknights/MaaAssistantArknights/commit/f90fc74916d9971922f4430e1bbb580a548dbdd1)
